x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_analytics.tsx

```diff
@@ -0,0 +1,206 @@
+  const {
+    setNoResultsCardVisible,
+    setQueriesCardVisible,
+    setStartDate,
+    setEndDate,
+    setSearchSessionId,
+    setTimeChange,
+  } = useActions(EngineAnalyticsLogic);
+
+  const { xyAttributes, metricAttributesQueries, metricAttributesNoResults } =
+    useValues(EngineAnalyticsLensLogic);
+  const { getLensAreaAttributes, getLensMetricAttributes } = useActions(EngineAnalyticsLensLogic);
+
+  const { data, defaultDataView, formula } = useValues(KibanaLogic);
+
+  useEffect(() => {
+    setSearchSessionId(data?.search?.session.start());
+    getLensMetricAttributes({
+      defaultDataView,
+      formula,
+      isNoResultsCardVisible,
+    });
+  }, []);
+  useEffect(() => {
+    getLensAreaAttributes({
+      defaultDataView,
+      formula,
+      isNoResultsCardVisible,
+    });
+  }, [isNoResultsCardVisible]);
+
+  useEffect(() => {
+    setStartDate(time.from);
+    setEndDate(time.to);
+  }, [time.from, time.to]);
+
+  return (
+    <EuiFlexGroup wrap direction="column">
+      <EuiFlexGroup wrap>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiFlexGroup gutterSize="m">
+              <EuiIcon size="l" type="search" />
+              <EuiText size="s">
+                <h2>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.engine.overview.analytics.Title',
+                    {
+                      defaultMessage: 'Analytics',
+                    }
+                  )}
+                </h2>
+              </EuiText>
+            </EuiFlexGroup>
+            <EuiSpacer size="s" />
+            <EuiFlexGroup wrap>
+              <EuiFlexItem grow>
+                <EuiText color={colors.subduedText} size="s">
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.engine.overview.analytics.subTitle',
+                    {
+                      defaultMessage:
+                        'Gain insight into query frequency, how many queries returned no results, and how many clicks they have generated.',
+                    }
+                  )}
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmptyTo
+                  iconType="eye"
+                  onClick={() => {
+                    KibanaLogic.values.navigateToUrl(ANALYTICS_PLUGIN.URL, {
+                      shouldNotCreateHref: true,
+                    });
+                  }}
+                  to={''}
+                >
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.engine.overview.analytics.button.buttonLabel.viewAnalytics',
+                    {
+                      defaultMessage: 'View Analytics',
+                    }
+                  )}
+                </EuiButtonEmptyTo>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexGroup>
+      <div hidden>
```

Comment:
We would want to call the lens component for getting the Metric component values but not display it. I initially tried to remove `hidden` but it would still show the empty space in UI.